Task4/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="css/content.css" rel="stylesheet" type="text/css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Task2</title>
+</head>
+<body>
+<h1 id="navigation">Chapters</h1>
+    <nav class="navigation">
+        <ul>
+            <li><a href="#a1">What is Lorem ipsum</a></li>
+            <li><a href="#a2">Why do we us it</a></li>
+            <li><a href="#a3">Where does it come from?</a></li>
+            <li><a href="#a4">Where can I get some?</a></li>
+            <li><a href="#a5">There standards</a></li>
+            <li><a href="#a6">Section</a></li>
+            <li><a href="#a7">1914</a></li>
+            <li><a href="#a8">Section</a></li>
+            <li><a href="#a10">Donate</a></li>
+            <li><a href="https://www.google.com" target="_blank">GOOGLE</a></li>
+        </ul>
+    </nav>
+    <div class="moving">
+<video width="400" height="300" controls="controls">
+    <source src="video/videoplayback%20(1).mp4" type="video/mp4" codecs="avc1.42E01E, mp4a.40.2">
+</video>
```

Comment:
Please keep the level of nesting tags (no spaces before tag <video>). Also review all your code, it contains a huge amount of the same "mistakes"

Reply:
Where did you read about attribute `codecs` and what it do? I can't find information about it?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source

Reply:
Why did you define the value of `controls` attribute? It's not a mistake but there are no sense.
Read more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video)